src/app/api/dashboard/probo-insights/route.ts

```
@@ -1,92 +1,129 @@
-      { success: false, error: 'Probo integration not yet implemented' },
-      { status: 501 }
-    );
+    const validatedData = ActionSchema.parse(body);
+
+    switch (validatedData.action) {
+      case 'setup':
+        const integration = await EnhancedProboService.setupIntegration(
+          organization.id,
+          validatedData.apiKey,
+          validatedData.webhookUrl
+        );
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)